### DISCUSSION TITLE
LS-GL no SSH connection

### DISCUSSION THREAD

**Discussion by Enonir** [1000001101000/Debian_on_Buffalo#152] [Q&A] [answered]
Hello!

New hard drive, after succseful TFTP boot I can not connect via SSH - using Putty or Windows command line. Always Connection refused. Is there another SSH port then 22? Checked images: Bullseye Buster Stretch.

**Comment by 1000001101000** [comment, marked as answer]
To start ssh the installer connects to a Debian mirror to download and install some needed packages. If the network you have setup to perform the TFTP boot doesn’t also provide DHCP, DNS and a path to the internet it will fail to start SSH.

**Reply by Enonir** [reply]
Thank you! Works like a charm.
